File: AjeshKumarS/cats-and-a-mouse.py
#!/bin/python3

import sys


q = int(input().strip())
for i in range(q):
    x,y,z = input().strip().split(' ')
    x,y,z = [int(x), int(y), int(z)]
    if(abs(z-x) < abs(z-y)):
        print("Cat A")
    elif(abs(z-x) > abs(z-y)):
        print("Cat B")
    else:
        print("Mouse C")

File: AshmiFathima/CatsandaMouse.cpp
#include <cmath>

#include <iostream>


using namespace std;


int main(){
    int q;
    cin >> q;
    for(int a0 = 0; a0 < q; a0++){
        int x;
        int y;
        int z;
        cin >> x >> y >> z;
        if(abs(x-z)==abs(y-z))
            cout<<"Mouse C"<<endl;
        else if(abs(x-z)<abs(y-z))
            cout<<"Cat A"<<endl;
        else
            cout<<"Cat B"<<endl;
    }
    return 0;
}
